### DISCUSSION TITLE
Game Crash

### DISCUSSION THREAD

**Discussion by Squander12** [MCPHackers/RetroMCP-Java#108] [Q&A]
I decided to fix the skins in the 1.7.3 beta, when I recompile the code and put it into the beta jar file and try to run the game, it crashes, exit code 1. I changed the code in classes, dc, gs, xz. So EntityPlayerSP, EntityPlayer and in EntityOtherMP, if I'm not mistaken, when I upload the last EntityOtherMP the game works. But when I add the rest then it crashes.

**Comment by PhoenixVX** [comment]
You'll need to link the crash log to get any help and the relevant code according to the stacktrace provided by the crash log.

**Reply by Squander12** [reply]
I already fixed it, I had to remove META-INF
